s3torchconnector/src/s3torchconnector/_s3client/_s3client.py

```diff
@@ -5,9 +5,7 @@
 from functools import partial
 from typing import Optional, Any
 
-from .s3reader import S3Reader
-from .s3writer import S3Writer
-
+from s3torchconnector import S3Reader, S3Writer
```

Comment:
do we use s3torchconnector or .?

Reply:
They are moved to the main module now, that's why we use `s3torchconnector`.